recipes/galaxy-containers/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "21.1.0" %}
+{% set sha256 = "51a3a9213710cf22d32996271b34d745e34fac78e51a7398b530f108df42a9d5" %}
+{% set galaxy_version = version.split(".")[:2]|join(".") %}
+
+package:
+  name: galaxy-containers
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/g/galaxy-containers/galaxy-containers-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    # Depend on galaxy-util during build time to avoid packaging galaxy/__init__.py .
+    - galaxy-util >={{ galaxy_version }}
+  run:
+    - galaxy-util >={{ galaxy_version }}
+    - python >=3.6
+    - requests
+    # extra: docker
+    # - docker
+
+test:
+  imports:
+    - galaxy
+    - galaxy.containers
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://galaxyproject.org
+  license: AFL-3.0
+  license_file: LICENSE
+  summary: Galaxy Container Modeling and Interaction Abstractions
+  doc_url: https://docs.galaxyproject.org
+  dev_url: https://github.com/galaxyproject/galaxy
+
+extra:
+  identifiers:
+    - biotools:galaxy
+    - doi:10.1093/nar/gky379
```

Comment:
This section is bioconda-specific, IIRC.
```suggestion
```

Reply:
:-(

Reply:
I know! But the linter complains...